save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt

```diff
@@ -124,6 +131,45 @@ class ExecutionController(private val executionService: ExecutionService,
                     }
                 }
 
+    /**
+     * Delete all executions by project name and project owner
+     *
+     * @param name name of project
+     * @param owner owner of project
+     * @return ResponseEntity
+     * @throws ResponseStatusException
+     */
+    @PostMapping("/deleteAllExecution")
+    fun deleteExecutionForProject(@RequestParam name: String, @RequestParam owner: String): ResponseEntity<String>? {
+        val projectId = projectService.findByNameAndOwner(name, owner)?.id
+
+        try {
+            testExecutionService.deleteTestExecutionWithProjectId(projectId)
+            agentService.deleteAgentWithProjectId(projectId)
```

Comment:
Also need to delete agent statuses?

Reply:
Yes. And need add foreign key on agent_id in agent_status

Reply:
Added